src/App.js

```diff
@@ -1,23 +1,18 @@
+import { fetch } from 'whatwg-fetch';
```

Comment:
use this style 
```javascript
import 'whatwg-fetch'
```
https://www.npmjs.com/package/whatwg-fetch#importing 

to polyfill it by default